changelog.md

```diff
@@ -2,6 +2,7 @@
 
 ### [Latest]
 
+- Rename flavour_categories -> flavours [#146](https://github.com/umami-hep/puma/pull/149)
```

Comment:
The PR number and link is wrong. Should be 154

Reply:
thanks! resolved